corehq/apps/accounting/migrations/0043_add_web_user_to_payment_methods.py

```diff
@@ -0,0 +1,317 @@
+# -*- coding: utf-8 -*-
+from south.db import db
+from south.v2 import DataMigration
+from django.db import connection
+
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        sql = """
+            UPDATE accounting_paymentmethod
```

Comment:
pretty unusual to see raw SQl in hq code.  any particular reason you preferred this to using the django orm?